.github/workflows/build_linux.yml

```diff
@@ -306,10 +306,10 @@ jobs:
       always() && !cancelled() &&
       (needs.handle-syncwith.result == 'success' || needs.handle-syncwith.result == 'skipped') &&
       needs.build-and-test-linux.result == 'success'
-    uses: NilFoundation/evm-placeholder-verification/.github/workflows/reusable-verify-proofs.yml@5bb7c4b0790f56b3a339e33ba442816fac17ab88
+    uses: NilFoundation/evm-placeholder-verification/.github/workflows/reusable-verify-proofs.yml@2a0ef5fc67e97be8e3c7b59338cf9eecd030c57d
```

Comment:
fyi: no need to update workflow reference if we didn't modify workflow file. Updating ref in `with:` section is enough

Reply:
I guess better use same revision for `workflow` and `code` references

Reply:
If we use the same refs, it is harder to establish which commit affects workflow code. No need to revert, just my thoughts